src/ray/raylet/runtime_env_agent_client.cc

```diff
@@ -151,7 +151,7 @@ class Session : public std::enable_shared_from_this<Session> {
 
   void on_connect(beast::error_code ec, tcp::resolver::results_type::endpoint_type) {
     if (ec) {
-      Failed(ray::Status::NotFound("on_connect " + ec.message()));
```

Comment:
The implementation creates multiple temporary strings, `absl::StrCat` only creates one.